src-docs/src/views/guidelines/forms.js

```diff
@@ -78,14 +79,13 @@ export default () => (
 
     <GuideRule
       heading="Provide transparency"
-      description="It should be clear to the user what each part of the form is meant to accomplish. ">
+      description="It should be clear users what they can accomplish in each part of the form. ">
```

Comment:
Missing a `to` here?
```suggestion
      description="It should be clear to users what they can accomplish in each part of the form. ">
```